drivers/hmis/app/graphql/types/base_enum.rb

```diff
@@ -55,6 +55,8 @@ def self.hud_enum(hash)
     end
 
     def self.value_for(key)
+      raise "Unrecognized key '#{key}' for enum #{name}" unless values[key].present?
```

Comment:
Maybe
```suggestion
      raise "Unrecognized key '#{key}' for enum #{name}" unless values.has_key?(key)
```
In case we end up with a case where the value for an enum is supposed to be `nil`?

Reply:
Good idea, fixed